feature/bgp/addpath/ate_tests/addpath/README.md

```diff
@@ -0,0 +1,91 @@
+## Summary
+This test checks for DUT functionality of BGP addpath capability. It also confirms accuracy of addpath capabilities negotiation, following checks
+1. DUT sets value for the Send/Receive field as "3" during capabilities negotiation to signal support for both Send and Receive ability for multiple paths.
+2. DUT sets value for the Send/Receive field as "1" during capabilities negotiation to signal support for Receive ability for multiple paths.
+3. DUT sets value for the Send/Receive field as "2" during capabilities negotiation to signal support for Send ability for multiple paths.
+4. Tests are conducted at scale of 1M v4 and 600k v6 routes with 64 different NHs each. Also, routes are withdrawn at scale to ensure addpath functionality is maintained at scale. Multipath is also enabled on the DUT to extend addpath functionalities implications on FIB programming at scale. 
+
+## Topology:
+ATE1:Port1 <-EBGP-> DUT:Port1
+ATE2:Port2 <-EBGP-> DUT:Port2
+ATE3:Port3 <-IBGP-> DUT:Port3
+ATE4:Port4 <-IBGP-> DUT:Port4
+ATE5:Port5 <-IBGP-> DUT:Port5
```

Comment:
This do not renedr well in markdow. You my like to use "\" at the end of line.
And firthermore intention for this section is to indicate what testbed topology is  - one of filenems `*.testbed` from 
https://github.com/openconfig/featureprofiles/tree/main/topologies

In this case [atedut_9.testbed](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_9.testbed)

Reply:
ack, addressed